README.md

```diff
@@ -67,8 +66,14 @@ compiler available. On Debian-based distros this usually means installing the
 2. Install the packages required for development:
    `make pip-sync`
 
-3. That's basically it. You should now be able to run the test suite:
-   `make test`.
+3. Compile the C extensions in `streaming_form_data`:
+   `make compile`
```

Comment:
The C file is also checked in as part of the source code so this step wouldn't really change anything. I wonder if there's a different reason you added this step here? 🤔

Reply:
You're right, I forgot about that, sorry! I just wanted to clarify the instructions for those without much Cython experience (like me 😅). I reworded it as a separate note in 5a78097, but I'm also happy to take it out altogether if you prefer.